codex-ui/src/vue/components/menu/ContextMenu.vue

```diff
@@ -0,0 +1,116 @@
+<template>
+  <div :class="$style['context_menu']">
+    <div
+      v-if="showSearch"
+      :class="$style['context_menu__search']"
+    >
+      <Input :class="$style['context_menu__input']" />
+      <div :class="$style['context_menu__sep-item']">
+        <div :class="$style['context_menu__separator']" />
+      </div>
+    </div>
+    <div :class="$style['context_menu__scrollable']">
+      <template
+        v-for="(item, index) in items"
+        :key="index"
+      >
+        <div
+          v-if="item.type === 'default'"
+          :class="[$style['context_menu__def-item']]"
+        >
+          <div :class="$style['context_menu__body']">
+            <Icon
+              v-if="item.icon !== undefined"
+              :name="item.icon"
+            />
+            <div>
+              {{ item.title }}
+            </div>
+          </div>
+        </div>
+        <div
+          v-if="item.type === 'separator'"
+          :class="$style['context_menu__sep-item']"
```

Comment:
class name is not meaningful. Maybe call wrapper element a 'context-menu__separator' and inner element 'context-menu__line'?